Ginger/Unit Tests/NonUITests/AutoPilot/POMLearningTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using GingerCore;
+using GingerCore.Drivers;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.UIElement;
+using GingerCore.Actions.Common;
+using GingerCore.Actions;
+using System.Threading.Tasks;
+
+namespace UnitTests.NonUITests.AutoPilot
+{
+    /// <summary>
+    /// Summary description for POMLearning
+    /// </summary>
+    [TestClass]
+    public class POMLearningTests
+    {
+
+        [TestMethod]
+        public void POMLearnFacebookPageTest()
+        {
+            //Arrange
+            ObservableList<ElementInfo> mElementsList = new ObservableList<ElementInfo>();
+            SeleniumDriver seleniumDriver = new SeleniumDriver(GingerCore.Drivers.SeleniumDriver.eBrowserType.Chrome);
+            seleniumDriver.StartDriver();
+            seleniumDriver.RunAction(new ActBrowserElement() { ControlAction = ActBrowserElement.eControlAction.GotoURL, ValueForDriver = "http://www.facebook.com" });
+            //Act
```

Comment:
Please learn static HTML file added to resources (same like we do in Web unit tests).
then check:
- actual amount of learned elements
- set test for filters and see that you get what expected in Mapped and un mapped from count and values
- set test for Locators details for 2-3 elelements from diffrent types 
- set test for properties values for 2-3 elements from diffrent types
- set tests for LocateElement by multi locators
- set tests for locate element by POM